MaxText/configs/base.yml

```diff
@@ -271,9 +271,17 @@ inference_microbenchmark_prefill_lengths: "64,128,256,512,1024"
 inference_microbenchmark_stages: "prefill,generate"
 inference_microbenchmark_loop_iters: 10
 inference_microbenchmark_log_file_path: ""
+inference_microbenchmark_flatten_results: False
 
 key_value_axis_order_product_id: ""
 prefill_key_axis_order: "1,2,0,3"
 prefill_value_axis_order: "1,2,0,3"
 ar_key_axis_order: "1,2,0,3"
-ar_value_axis_order: "1,2,0,3"
\ No newline at end of file
+ar_value_axis_order: "1,2,0,3"
+
+# Lists of KV-related settings over which to loop.
+# Each iteration will take same index from every list; therefore lists must be of the same length
+inference_microbenchmark_sweep_additional_metadata: ""
```

Comment:
I'd suggest remove this, as following below files, you should be able to get all the config from inside the inference_microbenchmark.py

https://github.com/google/maxtext/blob/main/MaxText/configs/base.yml#L58
-> https://github.com/google/maxtext/blob/main/MaxText/pyconfig.py#L265
-> https://github.com/google/maxtext/blob/main/MaxText/max_utils.py#L165